modules/core/src/main/resources/default.scala-steward.conf

```diff
@@ -0,0 +1,59 @@
+updates.ignore = [
+  // Ignore the next Scala 3 version until it is announced.
+  { groupId = "org.scala-lang", artifactId = "scala3-compiler",      version = { exact = "3.1.1" } },
+  { groupId = "org.scala-lang", artifactId = "scala3-library",       version = { exact = "3.1.1" } },
+  { groupId = "org.scala-lang", artifactId = "scala3-library_sjs1",  version = { exact = "3.1.1" } },
```

Comment:
@Kordyjan @romanowski FYI: Here are the version numbers that need to be bumped once 3.1.1 is announced.

Btw, is 3.1.1 the next Scala 3 version that will be released? I'm asking because a 3.1.2-RC version seems to be already available.

Reply:
I think you can see nightly builds of the compiler as 3.1.2-RC, as 3.1.2-RC1 will be the next release from the master branch. 3.1.1 is right now after the first RC and lives on `release-3.1.1` branch.